src/components/editor/editor-pane/tool-bar/utils/toolbarButtonUtils.ts

```diff
@@ -29,7 +29,15 @@ export const addImage = (editor: Editor): void => addLink(editor, '!')
 export const addLine = (editor: Editor): void => changeLines(editor, line => `${line}\n----`)
 export const addCollapsableBlock = (editor: Editor): void => changeLines(editor, line => `${line}\n<details>\n  <summary>Toggle label</summary>\n  Toggled content\n</details>`)
 export const addComment = (editor: Editor): void => changeLines(editor, line => `${line}\n> []`)
-export const addTable = (editor: Editor): void => changeLines(editor, line => `${line}\n| # 1  | # 2  | # 3  |\n| ---- | ---- | ---- |\n| Text | Text | Text |`)
+export const addTable = (editor: Editor, rows: number, columns: number): void => {
+  const rowArray = Array.from(Array(rows).keys())
```

Comment:
You could create a utility function for number ranges and replace 4 ugly array hacks with 4 utility calls.

Reply:
done